.github/workflows/build_docker_images.yml

```diff
@@ -44,3 +44,17 @@ jobs:
     - name: docker build images
       run: |
         DOCKER_BUILDKIT=1 docker build --file ./scripts/github_runner/Dockerfile.${{ matrix.image }} .
+  compose_images:
+    name: Build docker-compose function images
+    runs-on: ubuntu-18.04
+    strategy:
+      fail-fast: false
+      matrix:
+        image: [tests/producer-consumer]
+    steps:
+    - uses: actions/checkout@v2
+    - name: docker-compose build images
+      run: |
+        cd ./function-images/${{ matrix.image }}
```

Comment:
why is it here?

Reply:
docker-compose has to be run in the same directory as the docker-compose file. I searched about for a bit but I didn't find a way of providing the path as an argument

Reply:
oh unless you mean the entire thing? I can make a new workflow if that would fit in better?